### ISSUE TITLE
Mutilple update emails sent to admin

### ISSUE THREAD

**Issue by hughconway** [Adyen/adyen-magento#786]
Every time an order is placed a set of three order update messages are sent all at the same time to the admin copy email address.  The appear to be generated as a result of the status updates generated by Adyen as the payment moves from Pending to Processed.  These emails are not being sent to the customer.  I have "Customers receive order status update email (deprecated)." set to "No" as instructed by the Adyen Helpdesk.  I have "Capture Delay" set to "Immediate".

**Comment by rikterbeek** [comment]
Hi @hughconway,

It looks like you have a setting in magento turned on that you want to receive admin updates if there is a comment history created. If I understand correctly you do not want this mails be received by your admin email ? If that is the case you can disable this in the Magento configurations.

Regards,
Rik
Adyen

**Comment by hughconway** [comment]
Hi Rik,

I do want to receive order update emails but there seems to be a discrepancy as this setting is set to yes as per the attached screenshot but it is also sending the system generated order updates but only sending them to the BCC admin email address, not the customer email address.  This results in multiple status update emails being sent to the admin email address which is quite annoying for the sales team! The screenshot shows the customer is not notified so why is the admin copy email being sent?

Thanks,

Hugh
![screenshot 2017-07-19 14 36 05](https://user-images.githubusercontent.com/6165490/28369630-1cf66d62-6c90-11e7-86eb-12aecec3aee3.png)
![screenshot 2017-07-19 14 36 28](https://user-images.githubusercontent.com/6165490/28369631-1cfb14d4-6c90-11e7-9684-647ef1c89c9b.png)

**Comment by rikterbeek** [comment]
Hi @hughconway,

You have the setting to no so that is why it is not send to the customer. You have the order comments turned on no the screenshot so that is why it is send to your mail address with BCC.

Regards,
Rik
Adyen

**Comment by seansan** [comment]
@hughconway Did this solve your problem? (I remember we had this too when we started). The setting is a little confusing. We also had order comments turned on: but used it as
- if normal order then order order email
- if order has a comment then it will use the order comments flow

I am not sure Adyen is doing it correctly. The Order Comments has a different meaning. 

But we decided to avoid discussions and disable order comment emails alltogether
